### DISCUSSION TITLE
Why my all repositories show as a Dangerous site

### DISCUSSION THREAD

**Discussion by codewithsad** [community/community#125656] [Repositories]
### Select Topic Area

Bug

### Body

Why my all repositories show as a Dangerous site.

This is my Github Profile.
https://github.com/codewithsad

![Screenshot_80](https://github.com/community/community/assets/136996951/dfd8e77f-0445-411e-9734-627c70bea847)

**Comment by davevad93** [comment]
Hi @codewithsad ,

This usually happens when `Google` detects that a website may contain social engineering or phishing content, for example that happens with repos copying the design of popular webs like `Facebook`, `Amazon`, `Instagram`, `Twitter`, `Netflix` etcetera. If your pages have something that looks like those websites that's the reason why are flagged as `Dangerous site`. `GitHub` has nothing to do with this, as this is a `Google` security policy, for more information see the Google Safe Browsing documentation on [Social engineering (phishing and deceptive sites)](https://developers.google.com/search/docs/advanced/security/social-engineering?hl=en).
